plaid/accounts.go

```diff
@@ -28,30 +28,30 @@ type AccountNumber struct {
 	WireRouting string `json:"wire_routing"`
 }
 
-type accountRequestJsonOptions struct {
+type accountRequestOptions struct {
 	AccountIDs []string `json:"account_ids"`
 }
 
-type accountRequestJson struct {
+type accountRequest struct {
 	ClientID    string                    `json:"client_id"`
 	Secret      string                    `json:"secret"`
 	AccessToken string                    `json:"access_token"`
-	Options     accountRequestJsonOptions `json:"options,omitempty"`
+	Options     accountRequestOptions `json:"options,omitempty"`
 }
 
-type accountResponseJson struct {
+type AccountResponse struct {
 	APIResponse
 	Accounts []Account `json:"accounts"`
 }
 
 // Returns the real-time balance for each of an Item's accounts.
 // See https://plaid.com/docs/api/#balance.
-func (c *Client) GetBalances(accessToken string) (resp accountResponseJson, err error) {
+func (c *Client) GetBalances(accessToken string) (resp AccountResponse, err error) {
```

Comment:
we should have this be a separate type `BalancesResponse` so that it can diverge from `AccountResponse` in the future.

Reply:
⚡️